src/ndi-source.cpp

```diff
@@ -953,6 +961,14 @@ void ndi_source_thread_stop(ndi_source_t *s)
 			"'%s' ndi_source_thread_stop: Stopped A/V ndi_source_thread for NDI source '%s'",
 			obs_source_name, s->config.ndi_source_name);
 
+		// ISSUE: I am not 100% convinced that we want to do this **here**.
+		// It depends on how we phrase/word the UI text.
+		// Do we really want to blank the video here **after** it stops,
+		// or should we really be blanking the video just **before** it restarts?
+		// Why can't we just blank the video in the receiver thread?
+		// Isn't that what the code did before the changes in this area?
+		// Why was that not sufficient? What were the problems with that?
```

Comment:
Valid suggestion. IIRC on why this was moved here was because in previous commit this was keeping a black frame when the source was part of a scene in preview > but that was a bug as well.

Technically we should be okay to have this source texture "cleaned" when we restart the thread. - Will test and confirm